plugins/modules/nd_flow_rules.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicolg@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: nd_flow_rules
+version_added: "0.2.0"
+short_description: Manage Flow Rules
+description:
+- Manage VRF Flow Rules on Cisco Nexus Dashboard Insights (NDI).
+author:
+- Gaspard Micol (@gmicol)
+options:
+  insights_group:
+    description:
+    - The name of the Insights Group.
+    type: str
+    aliases: [ fab_name, ig_name ]
+  site_name:
+    description:
+    - The name of the ACI Fabric.
```

Comment:
Can you make this description consistent amongst the modules. Now we have aci fabric, assurance entity, site, and perhaps other naming used for the same thing.

Reply:
I don't know which one is accurate. I'll have a look.